src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig

```diff
@@ -24,6 +24,8 @@
  *#}
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
+{% set hooks = renderhooksarray('displayAdminProductsExtra', { 'id_product': id_product }) %}
```

Comment:
Should be inside the content block in case someone already overrides this template.